packages/apps/human-app/server/src/modules/jobs-discovery/jobs-discovery.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Headers, Query } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+import { JobsDiscoveryService } from './jobs-discovery.service';
+import {
+  JobsDiscoveryParamsCommand,
+  JobsDiscoveryParamsDto,
+  JobsDiscoveryResponse,
+} from './interfaces/jobs-discovery.interface';
+
+@Controller()
+export class JobsDiscoveryController {
+  constructor(
+    private readonly jobsDiscoveryService: JobsDiscoveryService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+
+  @ApiTags('Jobs-Discovery')
+  @Get('/jobs')
+  @ApiOperation({
+    summary:
+      'Retrieve a list of filtered available jobs for passed Exchange Oracle url',
+  })
+  public async getJobs(
```

Comment:
Not all the fields should be required
![image](https://github.com/humanprotocol/human-protocol/assets/61605646/4e87e23a-2318-46b0-ba38-8d89bbf1de05)